src/porepy/models/constitutive_laws.py

```diff
@@ -4164,7 +4239,7 @@ def elastic_normal_fracture_deformation(
 
         nd_vec_to_normal = self.normal_component(subdomains)
 
-        # The effective contact traction. Units: Pa = N/m^(nd-1)
+        # The scaled effective contact traction [-].
```

Comment:
This is something one either needs to get used to and internalize I guess, or it is clear from the code. Here, a inconsistent use of "scaled_" and no prefix is used. In the end, all quantities here non-dimensionalized. (I think I like "nondimensionalized" also better than "scaled").

Reply:
Do you mean in comments on variable/method names?